### ISSUE TITLE
Edit Selected Row "Accept" Function Fails to Save Edits

### ISSUE THREAD

**Issue by w-m-buchanan** [hankinsoft/SQLPro#998]
While editing fields in an SQL table (Edit selected row…), edits appear on screen as expected.  

However, when changes are submitted using Accept at the bottom of the form, no changes to the field’s content are accepted.  

I mean, my edited table is not updated and its changes are lost. They show on screen as expected during the contents edit,  but these changes are not entered into the mySQL db once I accept the changes.  Any changes I make disappear once I accept the changes.

Bottom line is the "Selected row editor function" does not work and the reason why is not clear.

Other functions, e.g. New Table, work as expected, but not "Edit selected row..."

**Comment by hankinsoft** [comment]
Issue has been closed. Please edit the initial post (or create a new issue) and follow the template. Once completed, the issue may be reopened
